main.js

```diff
@@ -180,10 +194,13 @@ const createWindow = () => {
 
     mainWindow.webContents.on(`ready-to-show`, () => {
       checkDependencies();
-      exec(`wmctrl -r "${mainWindow.getTitle()}" -b add,skip_taskbar`, (error) => {
-        if (error) {
-          log.error(`Failed to execute wmctrl: ${error.message}`);
-        }
+      // Lazy load child_process only when needed
```

Comment:
Why you changed the code you don't need it? As far as I understand your point to modification, you missed the signature of the windows build. Why you've modified all other things in worked codebase without proper tests?